tests/integration/startup/startup_int_test.go

```diff
@@ -263,13 +308,14 @@ var _ = Describe("startup tests", Ordered, func() {
 			Expect(testutil.K3sCleanup(-1, "")).To(Succeed())
 		})
 	})
-
 })
 
-var failed bool
-var _ = AfterEach(func() {
-	failed = failed || CurrentSpecReport().Failed()
-})
+var (
+	failed bool
+	_      = AfterEach(func() {
+		failed = failed || CurrentSpecReport().Failed()
+	})
+)
```

Comment:
nit: can you revert this. `AfterEach` is required to have a var because of how gomega is structured, but its not really a variable, it should be read as a function.